### PR TITLE
Integrate demo scripts from notebook conversion into testing infrastructure

### PR DESCRIPTION
**Pull request by sdrave**

### CODE REVIEW

**Comment by sdrave** [comment]
@pmli, we should at least parametrize the problem size to make testing the scripts faster ..

File: src/pymortests/demos.py [hunk 1]
```diff
@@ -115,16 +119,16 @@
...
-    + THERMALBLOCK_ARGS
-    + THERMALBLOCK_ADAPTIVE_ARGS
-    + THERMALBLOCK_SIMPLE_ARGS
-    + THERMALBLOCK_GUI_ARGS
-    + BURGERS_EI_ARGS
-    + PARABOLIC_MOR_ARGS
-    + SYS_MOR_ARGS
-    + HAPOD_ARGS
-    + FENICS_NONLINEAR_ARGS
+    # DISCRETIZATION_ARGS
```
Comment (renefritze) [inline src/pymortests/demos.py:122]: Looks like you forgot something in here.

**Review by renefritze — CHANGES_REQUESTED** [review]

**Comment by pmli** [comment]
> @pmli, we should at least parametrize the problem size to make testing the scripts faster ..

Sure, I can work on that. In another PR, I guess?

File: src/pymortests/demos.py (hunk 1 above)
Comment (sdrave) [reply src/pymortests/demos.py]: And I thought: 'Don't forget to remove the comments again.'

**Comment by sdrave** [comment]
> Sure, I can work on that. In another PR, I guess?

This one would be fine as well IMHO. Whatever you prefer ..

**Comment by pmli** [comment]
Ok, I've parameterized the demos and made some of them run faster. I would do further improvements in another PR.

BTW, does anybody have an idea how to time a script that is interrupted by `plt.show()`? Or should we remove those?

**Review by pmli — APPROVED** [review]

**Comment by renefritze** [comment]
> BTW, does anybody have an idea how to time a script that is interrupted by `plt.show()`? Or should we remove those?

Monkey patch `plt.show` to stop/start with blocking maybe? Be careful though, there's already monkeying happening in `_test_demo` to ensure tests don't hang.

**Review by renefritze — APPROVED** [review]

**Pull request merged**